docs/kpod-save.1.md

```diff
@@ -11,9 +11,8 @@ kpod-save - Save an image to docker-archive or oci
 [**--help**|**-h**]
 
 ## DESCRIPTION
-**kpod save** saves an image to either **docker-archive** on the loacl machine.
-**kpod save** writes to STDOUT by default and can be redirected to a file
-using the **output** flag. The **quiet** flag suppresses the output when set.
+**kpod save** saves an image to either **docker-archive** or **oci-archive
+on the loacl machine by setting the value of the **format** to either **docker-archive** or **oci**. **kpod save** writes to STDOUT by default and can be redirected to a file using the **output** flag. The **quiet** flag suppresses the output when set.
```

Comment:
`s/loacl/local/`.  Is it `oci` or `oci-archive`?

(And pedantically, this says that `kpod save` saves an image _by setting the value of an option_; perhaps `s/by setting the value of the **format** to/depending on the value of the **format** option, /`)

Reply:
fixed.